src/main/java/de/zalando/aruha/nakadi/domain/Cursor.java

```diff
@@ -8,6 +8,7 @@
 public class Cursor {
 
     public static final String BEFORE_OLDEST_OFFSET = "BEGIN";
+    public static final String AFTER_NEWEST_OFFSET = "END";
```

Comment:
+1
As we have exclusive offsets reading, it's not the one after newest, but the newest itself